server/app/com/xsn/explorer/tasks/MasternodeSynchronizerTask.scala

```diff
@@ -0,0 +1,72 @@
+package com.xsn.explorer.tasks
+
+import akka.actor.{Actor, ActorSystem, Props}
+import com.xsn.explorer.config.MasternodeSynchronizerConfig
+import com.xsn.explorer.models._
+import com.xsn.explorer.services.{XSNService}
+import javax.inject.Inject
+import org.scalactic.{Bad, Good}
+import org.slf4j.LoggerFactory
+import com.xsn.explorer.models.values.IPAddress
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+
+class MasternodeSynchronizerActor extends Actor {
+  import context._
+
+  def receive: Receive = {
+    val initialMasternodes = List[rpc.Masternode]()
+
+    behavior(initialMasternodes)
+  }
+
+  private def behavior(masternodes: List[rpc.Masternode]): Receive = {
+    case MasternodeSynchronizerActor.UpdateMasternodes(newMasternodes) =>
+      become(behavior(newMasternodes))
+    case MasternodeSynchronizerActor.GetMasternode(ipAddress) =>
+      sender() ! masternodes.find(x => x.ip.split(":")(0) == ipAddress.string)
+    case MasternodeSynchronizerActor.GetMasternodes =>
+      sender() ! masternodes
+  }
+}
+
+object MasternodeSynchronizerActor {
+  final case class UpdateMasternodes(masternodes: List[rpc.Masternode])
+  final case class GetMasternode(ipAddress: IPAddress)
+  final case class GetMasternodes()
+}
+
+class MasternodeSynchronizerTask @Inject()(
+    config: MasternodeSynchronizerConfig,
+    actorSystem: ActorSystem,
+    xsnService: XSNService
+)(implicit ec: ExecutionContext) {
+  private val logger = LoggerFactory.getLogger(this.getClass)
+
+  start()
+
+  @com.github.ghik.silencer.silent
+  def start() = {
+
+    if (config.enabled) {
+      logger.info("Starting masternode synchronizer task")
+
+      val masternodeSynchronizerActor = actorSystem.actorOf(Props[MasternodeSynchronizerActor], "masternode_synchronizer")
+
+      actorSystem.scheduler.schedule(config.initialDelay, config.interval) {
+        xsnService.getMasternodes().onComplete {
+          case Success(Good(masternodes)) =>
+            logger.info(s"Masternode information synced ${masternodes.length}")
+            masternodeSynchronizerActor ! MasternodeSynchronizerActor.UpdateMasternodes(masternodes)
+          case Success(Bad(error)) =>
+            logger.info(s"Masternode information syncronization failed due to $error")
+          case Failure(exception) =>
+            logger.info(s"Masternode information syncronization failed due to ${exception.getLocalizedMessage}", exception)
```

Comment:
use `getMessage` instead of `getLocalizedMessage`, we don't need to localize logs (localization isn't supported anyway), also, log a warning instead.